.gitignore

```diff
@@ -54,3 +54,7 @@ gradle-app.setting
 #Ignore examples auto generated files
 examples/**/Dependencies.toml
 examples/**/Config.toml
+
+# autogenerated files
+reflect-config.json
+resource-config.json
```

Comment:
Are these available in the latest master? AFAIK we removed these files.

Reply:
Oh, I was unaware, thank you for informing me. I'll update the git ignore now